frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json

```
@@ -0,0 +1,152 @@
+{
+  "Add": "Add",
+  "Pending Changes": "Pending Changes",
+  "The changes you've made required this virtual machine to be restarted.": "The changes you've made required this virtual machine to be restarted.",
```

This implies that the vm automatically restarts, and based on some of the other statements in this file it seems the user needs to restart for the changes to apply. I think it might make more sense to say:
```suggestion
  "The changes you made require you to restart this virtual machine..": "The changes you made require you to restart this virtual machine.",
```
or something like 
'You must restart your virtual machine to implement your changes."

---

Fixed!